core/network/impl/kademlia_value_storage.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "kademlia_value_storage.hpp"
```

Comment:
Full path please.

Reply:
Also, comments for StorageBackend apply here, too.

Reply:
Ouch! This is unused class of test implementation. Will be removed.